internal/worker/accounts.go

```diff
@@ -419,6 +420,15 @@ func (a *AccountMgr) refillAccount(ctx context.Context, contract api.ContractMet
 	return true, nil
 }
 
+func (a *Account) Token() rhpv4.AccountToken {
+	t := rhpv4.AccountToken{
+		Account:    rhpv4.Account(a.key.PublicKey()),
+		ValidUntil: time.Now().Add(time.Hour),
```

Comment:
We create them right before using them and they are the first thing verified by the host right? So I'd reduce this to 5 minutes at most.